packages/theme/src/components/Link.tsx

```diff
@@ -1,14 +1,12 @@
 import NextLink, { LinkProps as NextLinkProps } from 'next/link'
 import { AnchorHTMLAttributes } from 'react'
 
-import { buildShadow, buildTransition, useTheme } from '@edgeandnode/components'
-
-import { AppLocale, useI18n } from '@/i18n'
+import { buildShadow, buildTransition, useI18n, useTheme } from '@edgeandnode/components'
 
 export type LinkProps = Pick<NextLinkProps, 'replace' | 'scroll' | 'shallow' | 'prefetch'> &
   Omit<AnchorHTMLAttributes<HTMLAnchorElement>, 'href'> & {
     href?: NextLinkProps['href']
-    locale?: AppLocale
```

Comment:
Why remove all references to `@/i18n`?

Reply:
If anything, I think `@/i18n` should be part of the theme, and `website` should import it from the theme.

Reply:
I described https://github.com/graphprotocol/docs/pull/336#discussion_r1169369867 why I removed all `@/i18n` usages